master-firmware/src/robot_helpers/math_helpers.c

```diff
@@ -33,3 +33,16 @@ int math_clamp_value(int value, int min, int max)
         return value;
     }
 }
+
+int argmin(float* values, float num_values)
+{
+    int min_index = 0;
+    float min_value = 1e37;
```

Comment:
[nitpick] math.h has a macro `INFINITY`, which represents the largest value in a float.